### ISSUE TITLE
Login - Get Started tooltip shows again when existing users log in

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#48985]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.32-1
**Reproducible in staging?:** Y
**Reproducible in production?:** N/A
**Issue was found when executing this PR:** https://github.com/Expensify/App/pull/48963
**Email or phone of affected tester (no customers):** biruknew45+4375@gmail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Log in with a new account.
3. The "Get Started" tooltip appears. Tap on it and read the Concierge chat.
4. Create a workspace.
5. Log out.
6. Log back in with the same account.

## Expected Result:
The "Get Started" tooltip should only appear when logging in for the first time.

## Actual Result:
The Get Started tooltip appears even when the existing user logs in during step 6.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos
![Bug6599925_1726059708744!1](https://github.com/user-attachments/assets/b82f4a43-9e6e-405d-b185-edc8e6bc0ee6)

https://github.com/user-attachments/assets/82336c1a-e879-4f9e-8ca6-6d327deb8a1e

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by CyberAndrii** [comment]
This also happens on desktop Chrome and sometimes appears multiple times (no repro steps)

<img width="475" alt="Screenshot 2024-09-11 at 16 42 59" src="https://github.com/user-attachments/assets/3088ac45-7c35-43e7-bdf5-61c952326132">

**Comment by luacmartins** [comment]
I'm gonna demote it from being an App blocker since the fix seems to be in Auth and the bug is minor.

**Comment by zanyrenney** [comment]
does it need to stay hourly then @luacmartins ?

**Comment by NikkiWines** [comment]
I think this can probably be a daily, I'm looking into it today

**Comment by marcaaron** [comment]
>The "Get Started" tooltip should only appear when logging in for the first time.

Ah hmm I sort of disagree that this should be the expectation. The feature is intended to work like this:

- New user signs up
- Next time they re-authenticate we will set the nvp to false (which takes a couple of hours)

Is that so bad? It's not a normal experience to immediately sign out after you sign in so I don't feel strongly that we should optimize for this. I can't think of a great reason why it should work that way since this is only used to show a tool tip and it's probably "fine" to treat the user as "new" until some time passes and they re-auth.

**Comment by marcaaron** [comment]
The larger issue here might be that neither myself or @NikkiWines could get this tooltip to show up at all 😄

**Comment by NikkiWines** [comment]
I agree with @marcaaron here; it's not going to be the normal flow for a user to sign up and then immediately log back out - and even if they do, it's not very intrusive to just see the tooltips again.

I don't feel it's something we need to spend time modifying right now — though if we do end up wanting to change this, we can update the `isFirstTimeNewExpensifyUser` NVP to be set to `false` when the user logs out since that's an explicit action they'll take that stops them from being "new." 

Let's close for now and re-visit if/when this comes up again.
